src/Hocon.Configuration/HoconConfigurationFactory.cs

```diff
@@ -60,10 +59,29 @@ public static Config ParseString(string hocon)
         ///     The configuration defined in the configuration file. If the section
         ///     "akka" is not found, this returns an empty Config.
         /// </returns>
-        [Obsolete("Call the ConfigurationFactory.Default method instead.")]
         public static Config Load()
         {
-            return Default();
```

Comment:
Why this change?

Reply:
I was trying to follow Lightbend Hocon behaviour, will change it back.